src/components/blm/index.jsx

```diff
@@ -1,20 +1,14 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 
 // semantic-ui
-import {
-  Container,
-  Header
-} from 'semantic-ui-react';
+import { Container, Header } from "semantic-ui-react";
 
 // router
-import {
-  Switch,
-  Redirect,
-  Route,
-} from 'react-router-dom';
+import { Switch, Redirect, Route } from "react-router-dom";
```

Comment:
Small thing. Please see how we import other components and how they are multi-line if more than one component is imported.